react/src/page_template/template.tsx

```diff
@@ -30,6 +32,9 @@ export function PageTemplate({
 }): ReactNode {
     const [hamburger_open, set_hamburger_open] = useState(false)
     const [screenshot_mode, set_screenshot_mode] = useState(false)
+    const colors = useColors()
+
+    document.body.style.backgroundColor = colors.background
```

Comment:
This is probably fine but should really be a useEffect

Reply:
fixed